### PR TITLE
Enhance and cleanup code adding menu pages.

### PR DESCRIPTION
**Pull request by JoryHogeveen**
## Description

Just refactoring, no issue linked.

## ChangeLog

- None needed.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: classes/PodsAdmin.php [hunk 1]
```diff
@@ -433,49 +434,44 @@ public function admin_menu() {
 						continue;
 					}
 
+					$menu_location        = pods_v( 'menu_location', $pod['options'], 'default' );
```
Comment (tr1b0t) [inline classes/PodsAdmin.php]: :warning: Equals sign not aligned correctly; expected 1 space but found 8 spaces

<sub>hash: 74800499574bf36801abc331c697aa90</sub>

Comment (sc0ttkclark) [reply classes/PodsAdmin.php]:
```suggestion
					$menu_location = pods_v( 'menu_location', $pod['options'], 'default' );
```

**Comment by sc0ttkclark** [comment]
Ready for merge once you confirm there isn't anything further to be done here @JoryHogeveen

**Comment by JoryHogeveen** [comment]
If there is more I'll just create a new PR :)

**Pull request merged**